angular.json

```diff
@@ -114,9 +126,156 @@
                 }
               ],
               "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/occ/occ.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/occ/occ.environment.ccv2.ts"
+                },
+                {
+                  "replace": "projects/storefrontapp/src/environments/environment.ts",
+                  "with": "projects/storefrontapp/src/environments/environment.prod.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
+              "tsConfig": "projects/storefrontapp/tsconfig.app.prod.json"
+            },
+            "cds.development": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
+              "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/cds/cds.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/cds/cds.environment.dev.ts"
+                },
+                {
+                  "replace": "projects/storefrontapp/src/environments/occ/occ.environment.ts",
```

Comment:
Shouldn't we align with normal app that in development we use the occ.environment.ts file?

Reply:
We could do that. 
I was just thinking to make it more convenient to start CDS (in local) and to point to dev17 (as it's already configured).